devlib/target.py

```diff
@@ -1785,6 +1786,7 @@ def __init__(self,
                                             is_container=is_container,
                                             max_async=max_async)
         self.package_data_directory = package_data_directory
+        self.persistent_data_path = persistent_data_path
```

Comment:
I assume this should be the same for a ChomeOS target? In which case should we include a similar parameter for that target as well?

Reply:
Hm would it? I don't really know anything about ChromeOS from that side..

Reply:
For completeness the way we handle ChromeOS targets (which support Android apps) is essentially creating both a Linux and Android Target together so any parameters (or properties) we add should also be usable by the Android portion of the ChromeOs target.